zwave_js/CHANGELOG.md

```diff
@@ -1,5 +1,70 @@
 # Changelog
 
+## 0.1.85
+
+### New features
+
+- Z-Wave JS Server: Add support for inclusions that are started outside of Z-Wave JS Server
+- Z-Wave JS Server: Add support for lastSeen property
+- Z-Wave JS: Improve support for "identify" commands

```

Comment:
Likewise, has support for this landed in HA already?

Reply:
not yet